### PR TITLE
Add update-check sub-command to list current and latest version number

### LINKED ISSUE #2094: Expose update check
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): feature request

It would be useful for some downstream tooling to be able to script out checking for updates.  Possibly through a `minikube update-check` command.

Maybe something like this?
```
$ minikube update-check
CurrentVersion: v0.22.3
LatestVersion: v0.22.4

CurrentISOVersion: v0.23.5
LatestISOVersion: v0.23.5
```

cc @coollog @loosebazooka

### PR DESCRIPTION
**Pull request by mahdix**
Fixes #2094

### CODE REVIEW

**Comment by dlorenc** [comment]
@minikube-bot test this please

File: cmd/minikube/cmd/update-check.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	Short: "Print current and latest version number",
+	Long:  `Print current and latest version number`,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		// Explicitly disable update checking for the version command
+		enableUpdateNotification = false
+	},
+	Run: func(command *cobra.Command, args []string) {
+		url := constants.GithubMinikubeReleasesURL
+		r, err := notify.GetAllVersionsFromURL(url)
+		if err != nil {
```
Comment (r2d4) [inline cmd/minikube/cmd/update-check.go]: Maybe not the most idiomatic, but you might add a `|| len(r) < 1`, or more verbosely put it in its own conditional, so we absolutely guarantee the call to `r[0].Name` doesn't fail later.

This should also probably return an error code if we're unable to fetch the versions (or if we get an empty version list)

Comment (mahdix) [reply cmd/minikube/cmd/update-check.go]: I added a check for len of r and also exit code for both error types.

**Comment by r2d4** [comment]
@minikube-bot ok to test

**Pull request merged**